.github/workflows/daily-wpt-fyi.yml

```diff
@@ -0,0 +1,110 @@
+# This workflow runs every night and tests various versions of Node.js
+# (main branch build, current, and two latest LTS release lines) against the
+# `epochs/daily` branch of WPT.
+
+name: Daily WPT report
+
+on:
+  workflow_dispatch:
+  schedule:
+    # This is 20 minutes after `epochs/daily` branch is triggered to be created
```

Comment:
Is this fault-tolerant for the future?

Reply:
Even if it changed, it doesn't matter, we'd still execute daily on the current epochs/daily branch which only changes once per day.